src/main/java/org/bytedeco/javacpp/tools/Builder.java

```diff
@@ -476,13 +476,14 @@ File getOutputPath(Class[] classes, String[] sourcePrefixes) throws IOException
                 String resourceURL = Loader.findResource(classes[0], resourceName).toString();
                 String packageURI = resourceURL.substring(0, resourceURL.lastIndexOf('/') + 1);
                 for (int i = 1; i < classes.length; i++) {
-                    // Use shortest common package name among all classes as default output path
+                    // Use the longest common package name among all classes as default output path
                     String resourceName2 = '/' + classes[i].getName().replace('.', '/')  + ".class";
                     String resourceURL2 = Loader.findResource(classes[i], resourceName2).toString();
                     String packageURI2 = resourceURL2.substring(0, resourceURL2.lastIndexOf('/') + 1);
 
-                    String longest = packageURI2.length() > packageURI.length() ? packageURI2 : packageURI;
-                    String shortest = packageURI2.length() < packageURI.length() ? packageURI2 : packageURI;
+                    String longest, shortest;
+                    if (packageURI2.length() > packageURI.length()) { longest = packageURI2; shortest = packageURI; }
+                    else { longest = packageURI; shortest = packageURI2; }
```

Comment:
Let's just replace `>` with `>=` or `<` with `<=` instead?

Reply:
As you wish.